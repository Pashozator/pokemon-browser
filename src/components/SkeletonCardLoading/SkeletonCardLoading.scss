@import '../../styles/media-queries';

.skeleton-card-loading {
  margin: 0 0 10px 0;
  padding: 20px;
  width: 100%;
  height: 400px;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  display: flex;

  @include for-up-sm() {
    margin-right: 10px;
    width: 400px;
  }
}
