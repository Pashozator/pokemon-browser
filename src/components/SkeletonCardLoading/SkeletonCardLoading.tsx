import React from 'react';
import './SkeletonCardLoading.scss';
import Card from '../Card/Card';
import SkeletonLoading from '../SkeletonLoading/SkeletonLoading';

const SkeletonCardLoading: React.FC = props => {
	return (
		<Card className="skeleton-card-loading">
			<SkeletonLoading style={{ width: '100%', height: '100%' }} />
		</Card>
	);
};

export default SkeletonCardLoading;
