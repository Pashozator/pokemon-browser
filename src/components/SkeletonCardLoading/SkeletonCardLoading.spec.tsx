import React from 'react'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SkeletonCardLoading from './SkeletonCardLoading';

describe('SkeletonCardLoading component', () => {
	it('should render', () => {
		const wrapper = shallow(<SkeletonCardLoading/>);

		expect(wrapper.exists()).toBeTruthy();
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
