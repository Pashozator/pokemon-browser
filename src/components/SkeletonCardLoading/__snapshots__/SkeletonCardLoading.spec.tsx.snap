// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonCardLoading component should render 1`] = `
<Card
  className="skeleton-card-loading"
>
  <SkeletonLoading
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</Card>
`;
